dependencies64/ffmpeg/ffmpeg-build-linux/build.sh

```diff
@@ -448,3 +448,11 @@ CFLAGS="-I$TARGET_DIR/include -I$TARGET_DIR/usr/local/include" LDFLAGS="-L$TARGE
 	--enable-libv4l2 \
 	--enable-zlib
 make -j $jval && make install
+
```

Comment:
The lines below should be done manually by the developer IMHO. Consider that this build script will more often than not be used to upgrade ffmpeg, not only change build parameters. In that case the file names of the .so files will also change and need updating in modules/ffmpeg/CMakeLists.txt as well, so I think the copying/removing should be a manual step.